js/gated-content/src/components/event/EventTeaser.vue

```diff
@@ -68,22 +66,28 @@ export default {
   },
   computed: {
     date() {
-      return moment(this.video.attributes.date.value).format('YYYY-MM-DD');
+      return dayjs(new Date(this.video.attributes.date.value)).format('YYYY-MM-DD');
     },
     time() {
-      return moment(this.video.attributes.date.value).format('h:mm a');
+      return dayjs(new Date(this.video.attributes.date.value)).format('h:mm a');
     },
     duration() {
-      return moment.duration(moment(this.video.attributes.date.value)
-        .diff(moment(this.video.attributes.date.end_value))).humanize();
+      return `${Math.floor(dayjs.duration(
+        new Date(this.video.attributes.date.end_value) - new Date(this.video.attributes.date.value),
+      ).asMinutes())} minutes`;
     },
     startsIn() {
-      const duration = moment.duration(moment(this.video.attributes.date.value)
-        .diff(moment()));
-      if (duration.asHours() > 48) {
-        return duration.format('d [day]');
+      const eventStartDate = new Date(this.video.attributes.date.value);
+      const startsDuration = dayjs.duration(eventStartDate - new Date());
+
+      if (startsDuration.asHours() >= 48) {
+        return `${Math.floor(startsDuration.asDays())} days`;
       }
-      return duration.format('hh:mm:ss');
+
+      return `${(`0${Math.floor(startsDuration.asHours())}`)
```

Comment:
Won't linter allow us to have `.slice()` right after the expression it is applied to?